scripts/tests/chiptest/test_definition.py

```diff
@@ -283,7 +283,7 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                 # happen if the relevant application does not exist.  It's
                 # non-fatal as long as we are not trying to run any tests that
                 # need that application.
-                if len(path) == 1 and path[0] is None:
+                if path[-1] is None:
```

Comment:
What was the actual tuple for the failing path here? How did we get something where the "leaf" is None but the length is not 1?

Reply:
Ah, on Linux there are these `netns` prefixes on everything, I see.  Thank you!

Reply:
I wonder if we should not instead here make netns not create a list and have `None` preserved if the input path is None.

Even more, we should probably have `None` or `List of valid strings` rather than trying to accept `[None]` and detect it. We seem to introduce odd data along the pipeline and then try to deal with it. I would rather not have odd data to begin with.

Reply:
Doable, but requires finding all places that manipulate this path and fixing them....